zsh-wakatime.plugin.zsh

```diff
@@ -4,7 +4,11 @@
 send_wakatime_heartbeat() {
     entity=$(waka_filename);
     if [ "$entity" ]; then
-        (wakatime --write --plugin "zsh-wakatime/0.0.1" --entity-type app --project Terminal --entity "$entity"> /dev/null 2>&1 &)
+        if hash git>/dev/null 2>&1 && ${project=$(git rev-parse --show-toplevel>/dev/null 2>&1)}; then
+              (wakatime --write --project `basename $project` --plugin "zsh-wakatime/0.0.1" --entity-type app --entity "$entity"> /dev/null 2>&1 &)
```

Comment:
Cloud you extract the project name into a separate method? like the one for entity.